src/modules/monotype.js

```diff
@@ -34,47 +34,52 @@ webfont.modules.Monotype.NAME = 'monotype';
 webfont.modules.Monotype.HOOK = '__mti_fntLst';
 
 /**
- * __MonotypeAPIScript__ is the id of script added by google API. Currently 'webfonts.fonts.com' supports only one script in a page.
- * This may require change in future if 'webfonts.fonts.com' begins supporting multiple scripts per page.
+ * __MonotypeAPIScript__ is the id of script added by google API. Currently 'fonts.com' supports only one script in a page.
+ * This may require change in future if 'fonts.com' begins supporting multiple scripts per page.
  * @const
  */
 webfont.modules.Monotype.SCRIPTID = '__MonotypeAPIScript__';
 
-goog.scope(function () {
+goog.scope(function() {
   var Monotype = webfont.modules.Monotype,
-      Font = webfont.Font;
+    Font = webfont.Font;
 
-  Monotype.prototype.getScriptSrc = function (projectId, version) {
+  Monotype.prototype.getScriptSrc = function(projectId, version) {
     var p = this.domHelper_.getProtocol();
     var api = (this.configuration_['api'] || 'fast.fonts.net/jsapi').replace(/^.*http(s?):(\/\/)?/, "");
-    return p + "//" + api + '/' + projectId + '.js' + ( version ? '?v='+ version : '' );
+    return p + "//" + api + '/' + projectId + '.js' + (version ? '?v=' + version : '');
   };
 
-  Monotype.prototype.load = function (onReady) {
+  Monotype.prototype.load = function(onReady) {
     var self = this;
     var projectId = self.configuration_['projectId'];
     var version = self.configuration_['version'];
 
+    function checkAndLoadIfDownloaded() {
+      if (loadWindow[Monotype.HOOK + projectId]) {
+        var mti_fnts = loadWindow[Monotype.HOOK + projectId](),
+          fonts = [];
```

Comment:
No, despite the two space indentation we usually align variable names if used with the comma operator:

```
var one = 1,
    two = 2;
```